test/integration/puppetlabs/services/jruby/jruby_locking_test.clj

```diff
@@ -15,7 +15,16 @@
 (defn jruby-service-test-config
   [pool-size]
   (jruby-testutils/jruby-puppet-tk-config
-    (jruby-testutils/jruby-puppet-config {:max-active-instances pool-size})))
+    (jruby-testutils/jruby-puppet-config {:max-active-instances pool-size
+                                          :borrow-timeout 1})))
```

Comment:
is this milliseconds?  Is there any risk of setting it so low causing transient failures?  Or are we safe because we're never attempting multiple simultaneous borrows in this test?